odoo/addons/test_apikeys/static/tests/apikey_flow.js

```diff
@@ -6,7 +6,7 @@ const ajax = require('web.ajax');
 
 tour.register('apikeys_tour_setup', {
     test: true,
-    url: '/web',
+    url: '/web?debug=1',
```

Comment:
Probably used when working, but to remove ?

Reply:
Nope, as API key is now only available in debug mode, test whould be executed in debug mode to 'see' the API key section and interact with it.

Reply:
Ok, a note in code would be welcomed to know this is intended ^^